### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by ocrearyy**
Adding media queries for desktop version

### CODE REVIEW

**Review by prolajumokeoni — CHANGES_REQUESTED** [review]
Hi @oshanedesign 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! 
## Highlights 
- Good design ✔️ 
- No linter errors ✔️ 
- Github flow correctly used ✔️ 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -46,69 +52,84 @@ <h2 class="tonic">Tonic</h2>
...
                 <ul class="languages">
                     <li><button class="languages-List" type="submit">html</button></li>
                    <li><button class="languages-List" type="submit">css</button></li>
                     <li><button class="languages-List" type="submit">javascript</button></li>
                 </ul>
+                </div>
                 <button class="btn-Project" type="submit">See Project</button>
             </div>
+            </div>
             <div class="about-Myself">
```
Comment (prolajumokeoni) [inline index.html:128]: Your design             |  Figma Design
:-------------------------:|:-------------------------:
bad  | good



- [ ] Kindly align this div to the right so it looks like the original design, check the screenshot above 👍🏿 
- [ ] You can reduce the margin left to 3rem 👍🏿

Comment (oshanedesign) [reply index.html:128]: completed.

File: index.html [hunk 2]
```diff
@@ -14,24 +14,30 @@
...
                 <p>I am a software developer! I can help you build a product , feature or website Look through some of
                     my work and experience! If you like what you see and have a project you need coded, do not hestiate
                     to contact me.</p>
                 <h2 class="connect">LET'S CONNECT</h2>
                 <ul>
                     <li><img class="section1-Icons" src="images/social-media-icons.svg" alt="social-Media-Icons"></li>
                 </ul>
-            </div>
         </section>
         <section class="main-Grid-Works">
```
[the remark this answers is not in the document]
Comment (oshanedesign) [reply index.html:35]: completed

**Pull request merged**